### DISCUSSION TITLE
New projects with VS 17.10.1

### DISCUSSION THREAD

**Discussion by rvwilliams88** [unoplatform/uno#17009] [General]
I'm new to Uno Platform.  So far I've been using VS 2022 and following the tutorials such as SimpleCalculator without too many problems.  Somewhere along the way I upgraded to VS 2022 v17.10.1 Now when I start a new project using the Uno Platform App template the files such as MainPage and MainView are put into the Presentation folder instead of the root of the project and this gives them the wrong namespace.  What is going wrong here and how do I fix it?

**Comment by jeromelaban** [comment]
Thanks for the report.

There's nothing wrong with the files placed in the presentation folder. Why do you think this is incorrect?

**Reply by jeromelaban** [reply]
> Well, this wasn't what happened previously and it makes it more difficult to follow the tutorials.  The code set doesn't match up with the source saved on the GitHub repro. For example, I've been trying to follow the PeopleApp tutorial to learn more about MVUX and running the completed app just produces a blank screen.  I can't tell if this is because I've made a mistake with the code or because I had to move the MainPage,xaml and MainView.cs files out of the Presentation folder.

Thanks, that makes sense. The tutorials should indeed match the default template, but you should not have to move the files to make it work. /cc @agneszitte @dr1rrb

**Reply by agneszitte** [reply]
We will take a look at the turorials, thank you for the feedback! We will update with more info after that.
cc @eriklimakc, @nickrandolph

**Reply by agneszitte** [reply]
> Many apologies.  This is all due to an error on my part.  I was starting from the wrong template within the Uno Platform; I didn't choose the Blank template.  Sorry for causing unnecessary problems.

Thank you for the feedback @rvwilliams88, it is appreciated. We are currently working on improving the Uno Extensions documentation. If you have other uncertainties or doubts while going through the documentation, please don't hesitate to open other [separate discussions](https://github.com/unoplatform/uno/discussions) depending on the subject. But if you notice specific issues, don't hesitate to open [documentation issues](https://github.com/unoplatform/uno/issues/new/choose), please.

As you are new to Uno Platform, your feedback is important and helpful to us to improve the documentation overall. 
Thanks in advance ❤️

**Comment by rvwilliams88** [comment]
Hi Jerome

Thanks for the  response.

Well, this wasn't what happened previously and it makes it more difficult to follow the tutorials.  The code set doesn't match up with the source saved on the GitHub repro. For example, I've been trying to follow the PeopleApp tutorial to learn more about MVUX and running the completed app just produces a blank screen.  I can't tell if this is because I've made a mistake with the code or because I had to move the MainPage,xaml and MainView.cs files out of the Presentation folder.

Regards

Richard.
****************************************************
Richard Williams
email: ***@***.******@***.***>
mobile: 07981 734697
****************************************************

**Comment by rvwilliams88** [comment]
Many apologies.  This is all due to an error on my part.  I was starting from the wrong template within the Uno Platform; I didn't choose the Blank template.  Sorry for causing unnecessary problems.

Regards

Richard.
****************************************************
Richard Williams
email: ***@***.******@***.***>
mobile: 07981 734697
****************************************************
